frontend/static/style.css

```diff
@@ -45,9 +46,10 @@ button { padding: 10px; cursor: pointer; margin: 10px; }
 .labeled-play { display: inline-block; padding: 10px; border: 1px solid #000; }
 .labeled-play .label { text-align: center }
 .labeled-play .card { cursor: default; }
-.labeled-play.winning { background: #efefef; }
-.labeled-play.winning .card { background: #efefef; }
-.labeled-play.winning.landlord { background: #ddd; }
+.labeled-play.you {border: 3px solid #0575ff;}
+/* .labeled-play.winning { background: #efed9fa6; } */
```

Comment:
delete if no longer used?

Reply:
update as requested. I also removed '.you' from trick since u wanted player list to be on always (I wanted to hide it to have more screen space for showing more useful info like points, and previous trick)

There are a few UI improvements I'd like to see, but without a framework, it may be too much work. For example, the points can be just one line with details hidden, but can be expanded by individual user;  separation of trump suit from non trump suits in hand, etc. Without a css framework, it may be too much work

BTW, reset game should really be restart game since game setting is preserved